bin/dev

```diff
@@ -1,18 +1,5 @@
 #!/usr/bin/env node
-/* eslint-disable @typescript-eslint/no-var-requires */
-const path = require('path');
-
-const oclif = require('@oclif/core');
-
-const project = path.join(__dirname, '..', 'tsconfig.json');
-
-// In dev mode -> use ts-node and dev plugins
-process.env.NODE_ENV = 'development';
-
-require('ts-node').register({project});
-
-// In dev mode, always show stack traces
-oclif.settings.debug = true;
-
-// Start the CLI
-oclif.run().then(oclif.flush).catch(oclif.Errors.handle);
+(async () => {
+  const oclif = await import('@oclif/core');
+  await oclif.execute({ type: 'cjs', development: true, dir: __dirname });
+})();
```

Comment:
don't we need to handle error any more 🤔

Reply:
I just migrate that file following the guide. So I think it should be handled by the library